src/Microsoft.ML.Dnn/DnnCatalog.cs

```diff
@@ -97,23 +211,7 @@ public static class DnnCatalog
         /// <param name="labelColumnName">The name of the labels column.</param>
         /// <param name="scoreColumnName">The name of the output score column.</param>
         /// <param name="predictedLabelColumnName">The name of the output predicted label columns.</param>
-        /// <param name="arch">The architecture of the image recognition DNN model.</param>
-        /// <param name="epoch">Number of training iterations. Each iteration/epoch refers to one pass over the dataset.</param>
-        /// <param name="batchSize">The batch size for training.</param>
-        /// <param name="learningRate">The learning rate for training.</param>
-        /// <param name="disableEarlyStopping">Whether to disable use of early stopping technique. Training will go on for the full epoch count.</param>
-        /// <param name="earlyStopping">Early stopping technique parameters to be used to terminate training when training metric stops improving.</param>
-        /// <param name="metricsCallback">Callback for reporting model statistics during training phase.</param>
-        /// <param name="statisticFrequency">Indicates the frequency of epochs at which to report model statistics during training phase.</param>
-        /// <param name="framework">Indicates the choice of DNN training framework. Currently only tensorflow is supported.</param>
-        /// <param name="modelSavePath">Optional name of the path where a copy new graph should be saved. The graph will be saved as part of model.</param>
-        /// <param name="finalModelPrefix">The name of the prefix for the final mode and checkpoint files.</param>
         /// <param name="validationSet">Validation set.</param>
-        /// <param name="testOnTrainSet">Indicates to evaluate the model on train set after every epoch.</param>
-        /// <param name="reuseTrainSetBottleneckCachedValues">Indicates to not re-compute cached trainset bottleneck values if already available in the bin folder.</param>
-        /// <param name="reuseValidationSetBottleneckCachedValues">Indicates to not re-compute validataionset cached bottleneck validationset values if already available in the bin folder.</param>
-        /// <param name="trainSetBottleneckCachedValuesFilePath">Indicates the file path to store trainset bottleneck values for caching.</param>
-        /// <param name="validationSetBottleneckCachedValuesFilePath">Indicates the file path to store validationset bottleneck values for caching.</param>
         /// <remarks>
         /// The support for image classification is under preview.
```

Comment:
@codemzs - When should this `remark` get removed? I'm assuming we don't want to ship with this in the documentation.

Reply:
Removing the remark after talking to @codemzs offline. 

---
In reply to: [337567053](https://github.com/dotnet/machinelearning/pull/4362#discussion_r337567053) [](ancestors = 337567053)